src/automerge-action.ts

```diff
@@ -116,6 +116,17 @@ export class AutomergeAction {
       return false
     }
 
+    if (this.input.requiredLabels.length > 0) {
```

Comment:
I think this is unneeded, since the loop won't run anyways if it's empty.

Reply:
You're right, this was more for clarity, but will remove it.